README.md

```diff
@@ -48,3 +48,50 @@ We welcome feedback and suggestions from any Biodata Catalyst team on useful tes
+| Terra         | :heavy_multiplication_x: | :heavy_check_mark:       | :heavy_check_mark:       | :heavy_multiplication_x: | Future                   |      |
+| Gen3          | TBD                      | :heavy_multiplication_x: | TBD?                     | TBD?                     |                          |      |
+| Dockstore     |                          |                          | :heavy_multiplication_x: |                          |                          |      |
+| Seven Bridges | :heavy_multiplication_x: | TBD?                     | TBD?                     | :heavy_multiplication_x: | Future                   |      |
+| PIC-SURE      |                          | TBD?                     |                          |                          | :heavy_multiplication_x: |      |
+| HeLx          | TBD?                     | TBD?                     |                          | TBD?                     |                          | :heavy_multiplication_x: |
+
+ - :heavy_check_mark: : Implemented (probably not comprehensively; additional testing suggestions are very welcome)
+ - :heavy_multiplication_x: : Not Applicable
+ - TBD : To Be Done
+ - Future : Tests will be needed, but the features have not been built yet
+
+
+Current tests (all are run in each component's Staging environment):
+
+API Test 1:
+ - Create Terra workspace
+ - Import static PFB from Gen3 into workspace
+ - Check for success of PFB import into Terra workspace
+ - Delete workspace
+
+API Test 2:
+ - Import from Dockstore workflow to Terra workspace
+ - Check presence of Dockstore workflow in Terra workspace
+ - Delete Dockstore workflow from terra workspace
+
+API Test 3:
+ - Run DRS URI in md5sum workflow in Terra
+ - Check workflow run success
+ 
+ Currently tested endpoints are:
+
+ - https://us-central1-broad-dsde-alpha.cloudfunctions.net/martha_v2
+ - https://firecloud-orchestration.dsde-alpha.broadinstitute.org/status
+ - https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/importPFB
+ - https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/importPFB/{job_id}
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/submissions
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/methodconfigs
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/methodconfigs?allRepos=true
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/methodconfigs/{billing_project}/{workflow}
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/submissions/{submission_id}
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}
```

Comment:
Are any Gen3 endpoints tested?

Reply:
No, there were and credentialing is set up, but we only go from Terra to Gen3 and I'd like to see those tests if possible too.